semantics/common/configuration.k

```diff
@@ -101,6 +101,18 @@ module COMMON-CONFIGURATION
                          <access-specifier> .K </access-specifier>
                     </class>
                </classes>
+
+               <cpp-enums>
+                    <cppenum multiplicity="*" type="Map">
+                         <enum-id> 0 </enum-id>
```

Comment:
I would prefer to keep things consistent. Right now the keys for classes and namespaces are identifiers, having the keys for enums be integers introduces an inconsistency in the design. Is there a specific program you're trying to run that doesn't execute correctly with that solution?